storage-xray-udp/src/main/java/zipkin2/storage/xray_udp/UDPMessageEncoder.java

```diff
@@ -94,6 +94,10 @@ static void writeJson(Span span, Buffer buffer) throws IOException {
       }
     }
 
+    if (span.tags().get("xray.origin") != null) {
```

Comment:
maybe in the README make a table about data mapping? (in table form.. I'm lazy below)

name source destination notes
Origin span.tags[xray.origin] segment.origin link_to_docs

Reply:
Let's use `aws` prefix instead of `xray` similar to some of our other aws-specific tags

https://github.com/openzipkin/zipkin-aws/blob/master/brave-instrumentation/aws-java-sdk-v2-core/src/main/java/brave/instrumentation/awsv2/TracingExecutionInterceptor.java#L75